repos/fdbt-site/src/pages/createCaps.tsx

```
@@ -38,64 +59,254 @@ const CreateCaps = ({
+                                                        hideText
+                                                        addFormGroupError={false}
+                                                    >
+                                                        <input
+                                                            className="govuk-input govuk-input--width-4 govuk-currency-input__inner__input"
+                                                            name="capPrice"
+                                                            data-non-numeric
+                                                            type="text"
+                                                            aria-describedby="cap-price-hint"
+                                                            id="cap-price"
+                                                            defaultValue={
+                                                                userInput && userInput.cap.price
+                                                                    ? userInput.cap.price
+                                                                    : ''
+                                                            }
+                                                        />
+                                                    </FormElementWrapper>
+                                                </div>
+                                            </div>
+                                        </>
+                                    </FormGroupWrapper>
+                                </div>
+                                <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                                    <FormGroupWrapper
+                                        errors={errors}
+                                        errorIds={['cap-period-duration-quantity', 'cap-period-duration-unit']}
+                                        hideErrorBar
```

Do we need to hide the error bar here? Does it look ok with it?